bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -103,6 +103,54 @@ Builder withRequest(ValueProvider<ReadRowsRequest> request) {
       return this;
     }
 
+    /**
+     * Provides an updated request by adding a row range with start and end keys in the existing
```

Comment:
Please update docs to indicate that you are replacing the row set

Reply:
done.